resources/views/layouts/default/flights/simbrief_form.blade.php

```diff
@@ -1,269 +1,426 @@
+							<input type="hidden" id="type" name="type"  class="form-control" value="{{ $SimBriefType }}" maxlength="4" />
+						</div>
+						<div class="col-sm-4">
+							<label for="reg">Registration</label>
+							<input type="text" class="form-control" value="{{ $AcDetails->registration }}" maxlength="6" disabled/>
+							<input type="hidden" id="reg" name="reg" value="{{ $AcDetails->registration }}" />
+						</div>
+					</div>
+					<br>
+					</div>
+					
+					<div class="form-container-body">
+					<h6><i class="fas fa-info-circle"></i>&nbsp;@lang('pireps.flightinformations') for <b>{{ $flight->airline->icao }} {{ $flight->flight_number }}</b></h6>
+					<div class="row">
+						<div class="col-sm-4">
+							<label for="dorig">Departure Airport</label>
+							<input id="dorig" type="text" class="form-control" maxlength="4" value="{{ $flight->dpt_airport_id }}" disabled/>
+							<input id="orig" name="orig" type="hidden" maxlength="4" value="{{ $flight->dpt_airport_id }}" />
+						</div>
+						<div class="col-sm-4">
+							<label for="ddest">Arrival Airport</label>
+							<input id="ddest" type="text" class="form-control" maxlength="4" value="{{ $flight->arr_airport_id }}" disabled/>
+							<input id="dest" name="dest" type="hidden" maxlength="4" value="{{ $flight->arr_airport_id }}" />
+						</div>
+						<div class="col-sm-4">
+							<label for="altn">Alternate Airport</label> 
+							<input id="altn" name="altn" type="text" class="form-control" maxlength="4" value="{{ $ALTN }}" />
+ 						</div>
+					</div>
+					<br>
+					<div class="row">
+						<div class="col-sm-8">
+							<label for="route">Preferred Company Route</label>
+							<input id="route" name="route" type="text" class="form-control" placeholder="" maxlength="1000" value="{{ $flight->route }}" />
+						</div>
+						<div class="col-sm-4">
+							<label for="fl">Preferred Flight Level</label>
+							<input id="fl" name="fl" type="text" class="form-control" placeholder="" maxlength="5" value="{{ $flight->level }}" />
+						</div>
+					</div>
+					<br>
+					<div class="row">
+						<div class="col-sm-4">
+							<label for="std">Scheduled Departure Time (UTC)</label>
+							<input id="std" type="text" class="form-control" placeholder="" maxlength="4" value="{{ $flight->dpt_time }}" disabled/>
+						</div>
+						<div class="col-sm-4">
+							<label for="etd">Estimated Departure Time (UTC)</label>
+							<input id="etd" type="text" class="form-control" placeholder="" maxlength="4" disabled/>
+						</div>
+						<div class="col-sm-4">
+							<label for="dof">Date Of Flight (UTC)</label>
+							<input id="dof" type="text" class="form-control" placeholder="" maxlength="4" disabled/>
+						</div>
+					</div>
+					<br>
+					</div>
+					
+					<div class="form-container-body">
+					{{-- Get All Subfleets from flight and generate random load for each fare type of selected SubFleet --}}
+					@foreach($flight->subfleets as $SUB)
```

Comment:
Use `$subfleet`. Trying to keep these the same conventions/naming across the codebase

Reply:
Fixed.